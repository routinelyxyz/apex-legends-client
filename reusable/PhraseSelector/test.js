import React from 'react';
import { create } from "react-test-renderer";
import { PhraseSelector } from './';

test('it matches the snapshot', () => {
  const component = create(
    <PhraseSelector
      value="Rockalone"
      phrase="alone"
    />
  );
  expect(component.toJSON()).toMatchSnapshot();
    
  component.update(
    <PhraseSelector
      value="veryLongPhraseWith Some Text"
      phrase="longPhr"
    />
  );
  expect(component.toJSON()).toMatchSnapshot();
});