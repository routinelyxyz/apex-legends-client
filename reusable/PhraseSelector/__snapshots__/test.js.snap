// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches the snapshot 1`] = `
Array [
  "Rock",
  <strong
    className="selection"
  >
    alone
  </strong>,
  "",
]
`;

exports[`it matches the snapshot 2`] = `
Array [
  "very",
  <strong
    className="selection"
  >
    longPhr
  </strong>,
  "aseWith Some Text",
]
`;
